package.json

```diff
@@ -22,6 +22,7 @@
   "homepage": "https://github.com/brigade/eslint-config-brigade#readme",
   "dependencies": {
     "eslint-plugin-import": "^2.2.0",
+    "eslint-plugin-jsx-a11y": "^4.0.0",
```

Comment:
The commit message ("Add eslint-plugin-jsx-a11y") could explain what a11y is a little better, and what it means for us.